.idea/BE10-Basic.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Let;s schedule a meeting in order to clarify how not to add such files to the pull requests.